code/modules/mob/living/carbon/xenomorph/castes/wraith/abilities_wraith.dm

```diff
@@ -317,7 +329,11 @@ GLOBAL_LIST_INIT(wraith_banish_very_short_duration_list, typecacheof(list(
 		ghost.adjust_sunder(plasma_deficit)
 
 	starting_turf = null
-	add_cooldown()
+	plasma_cost = initial(plasma_cost) //Revert the plasma cost to its initial amount
+	var/cooldown_override
+	if(resync)
+		cooldown_override = 3 SECONDS //Partial cooldown if we cancelled out of Phase Shift with Resync
+	add_cooldown(cooldown_override)
```

Comment:
i don't know if i like this, 3 seconds cooldown is something very low and allows you to keep using phaseshift basically everytime until out of plasma.

Reply:
Every use has a wind up of 2 seconds; you can't just spam this to be invul mid combat or anything because you'll get killed in short order. What resync does is effectively give you a low cost scouting option.

Reply:
is the skill plasma use considerable at least ? i may let this pass if so

Reply:
It's currently 25 base and scales with the distance you travel; with Resync you don't travel any distance.